catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemValidator.java

```diff
@@ -26,35 +24,52 @@
  * required by the {@link ddf.catalog.content.StorageProvider}.
  */
 public class ContentItemValidator {
-
     private static final Pattern QUALIFIER_PATTERN = Pattern.compile("\\w[a-zA-Z0-9_\\-]+");
```

Comment:
One of the purposes for this change is because a URI is **too** broad which caused bugs. So we can't use URI. Also as previously stated the UUID will not work since we support no dashes. We really only support content:uuid#qualifier, so hence the regex to validate.